### ISSUE TITLE
Weights norm

### ISSUE THREAD

**Issue by oliviercailloux** [oliviercailloux/decision-uta-method#12]
decide whether you consider weights should sum to one and document and modify the code accordingly

**Comment by tonyseg** [comment]
It is the case at the beginning according to Labreuche paper.

**Comment by oliviercailloux** [comment]
Yes but we could be more general. Non-normalized weights can be nicer to use because the weights for example might be integers and mean something to the user (1 for normal criterion, 2 for “twice as important than normal”, …), and this meaning is much less clear after normalization.

Please, for each method in Tools which deals with weights, check whether you can make it work also when weights are not normalized (if it’s difficult, don’t do it), then document (in javadoc) whether the weights “must be normalized” or “must not necessarily be normalized”.
